xclim/indices/_anuclim.py

```diff
@@ -194,8 +196,9 @@ def precip_seasonality(
 @declare_units("[temperature]", tas="[temperature]")
 def tg_mean_warmcold_quarter(
     tas: xarray.DataArray,
-    op: str = None,
-    src_timestep: str = None,
+    *,
```

Comment:
Is there a consensus on not allowing positional arguments ?
It seems overkill for a function that takes only one dataarray argument.

Reply:
Woops, yes, we've discussed this and I think we deferred discussion on this change to much later (maybe v1). Will fix.